.travis.yml

```diff
@@ -39,7 +44,3 @@ script: ./travis.sh
 notifications:
   email:
     - cukes-devs@googlegroups.com
-
-branches:
-  only:
-    - master
```

Comment:
I think it would be better to separate this change and implement it for AppVeyor as well. As it's a trivial change and a PR wouldn't add anything to it, I'll push the change without PR.